lib/views/home/home.dart

```diff
@@ -52,47 +73,178 @@ class _HomeViewState extends State<HomeView> {
             data: Theme.of(context).copyWith(
               splashFactory: NoSplash.splashFactory,
             ),
-            child: SizedBox(
+            child: BottomAppBar(
+              padding: const EdgeInsets.symmetric(horizontal: 0),
               height: SizeSetter.getBottomNavigationBarHeight(),
-              child: BottomNavigationBar(
-                items: const [
-                  BottomNavigationBarItem(
-                    key: Key('home_view_button'),
-                    icon: Icon(Icons.home),
-                    label: 'Home',
-                  ),
-                  BottomNavigationBarItem(
-                    key: Key('wiki_view_button'),
-                    icon: Icon(Icons.book),
-                    label: 'Wiki',
-                  ),
-                  BottomNavigationBarItem(
-                    key: Key('profile_view_button'),
-                    icon: Icon(Icons.person),
-                    label: 'Profiel',
-                  ),
+              child: Row(
+                mainAxisSize: MainAxisSize.max,
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: <Widget>[
+                  _buildNavItem(Icons.book, "Wiki", 0),
+                  _buildNavItem(Icons.map, "Map", 1),
+                  _buildNavItem(Icons.add_box, "", -1, isCenter: true),
+                  _buildNavItem(Icons.history, "Meldingen", 2),
+                  _buildNavItem(Icons.person, "Account", 3),
                 ],
-                type: BottomNavigationBarType.fixed,
-                selectedItemColor: Theme.of(context).primaryColor,
-                unselectedItemColor: Colors.black,
-                backgroundColor: Colors.white,
-                selectedFontSize: SizeSetter.getBodySmallSize(),
-                unselectedFontSize: SizeSetter.getBodySmallSize(),
-                currentIndex: selectedIndex,
-                unselectedLabelStyle:
-                    CustomTheme(context).themeData.textTheme.bodySmall,
-                selectedLabelStyle:
-                    CustomTheme(context).themeData.textTheme.bodySmall,
-                onTap: onItemTapped,
               ),
             ),
           ),
           BottomNavigationBarIndicator(
-            selectedIndex: selectedIndex,
-            indicatorWidth: 55,
+            selectedIndex:
+                selectedIndex >= 2 ? selectedIndex + 1 : selectedIndex,
+            indicatorWidth: MediaQuery.of(context).size.width / 5,
           ),
         ],
       ),
     );
   }
+
+  Widget _buildNavItem(IconData icon, String label, int index,
```

Comment:
Ik zou dit soort code in het vervolg even in een andere file zetten. Dan blijft het iets overzichtelijker